### ISSUE TITLE
No google maps on show flight

### ISSUE THREAD

**Issue by CrOhN1** [leonardoxc/leonardoxc#62]
I have a wierd problem on a new leonardo instalation.

On show flight i get no google maps view. If i change the API Key to an incorrect one. The map is show momentarly, and then a error picture with "ups" javascript error.

What i am doing wrong?

**Comment by Mauroescobar** [comment]
In a last 15 days i was trying to display recent uploaded tracks to Paraglidingforum/Leonardo in a Google Earth: IGC2KMZ NEW option (Most detailed, bigger size), and the file size appear "0" cero!!! and Google Earth show correctly the right specific place but without any track draw.

The other two Google Earth viewing options shown correctly the track draw, but unfortunately the IGC2KMZ bring to me all the information that i want, do you know the solution? how can i do?

**Comment by CrOhN1** [comment]
No, i'm afraid not. Other sites i could check, the same is happening.

Abou my problem. Do you have any idea?

**Comment by Mauroescobar** [comment]
Thanks for your reply CrOhN1, i don't have answer for your problem, also i am in the same situation and use your chat to ask for obtain any answer from another user or Leonardo account manager, we need to wait that somebody look this chat and try to help us.

**Comment by CrOhN1** [comment]
Hi, do yuo have your instalation working correctly? What version are you using?
